melodic-manatees/early_internet/background_app/background_utility.py

```diff
@@ -0,0 +1,36 @@
+from PIL import Image
+import os.path
+from django.core.exceptions import ValidationError
+
+
+def resolution_checker(image):
+    '''check resolution of uploaded images, reject if too small or too big'''
+    im = Image.open(image)
+    width, height = im.size
+    size_in_bytes = os.stat(image).st_size
+    if width < 1200 or height < 800:
+        return False
+    if size_in_bytes > 2_621_400:
```

Comment:
I love that you're using these optional underscores to make the numbers more readable. However, it's not clear from this code _why_ you're checking this particular number. That makes this a so-called "magic number", which is something you should try to avoid.

In this case, it might be better to define this number as a constant at the top of the file, giving it a descriptive variable name like `MAX_SUPPORTED_SIZE` or something like that, so that this line could read `if size_in_bytes > MAX_SUPPORTED_SIZE`, which would be more expressive.